app/view/LayerTree.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * A Layertree which autoconnects to the map of the application.
+ */
+Ext.define('CpsiMapview.view.LayerTree', {
```

Comment:
Should  we place this in a separate folder/package?

Reply:
I wasn't sure. right now we only have on tree…